test/mocks/MockAppConfig.scala

```
@@ -65,7 +65,8 @@ class MockAppConfig(implicit val runModeConfiguration: Configuration) extends Ap
   override val deregisterForVat: String = "ye-olde-deregister-url"
   override val feedbackUrl: String = "/feedback"
   override val vatCorrespondenceChangeEmailUrl: String = "mock-change-email-url"
-  override val vatCorrespondenceChangePhoneNumbersUrl: String = "mock-change-phone-url"
+  override val vatCorrespondenceChangeLandlineNumberUrl: String = "mock-change-phone-url"
+  override val vatCorrespondenceChangeMobileNumberUrl: String = "mock-change-phone-url"
```

Could you make these mock values different from each other, just to avoid the possibility that one is used incorrectly in place of the other and the tests fail to pick it up.